cloudify_agent/api/pm/base.py

```diff
@@ -217,10 +226,14 @@ def __init__(self, logger=None, **params):
             'name') or self._get_name_from_manager()
         self.queue = params.get(
             'queue') or self._get_queue_from_manager()
-        self.broker_url = params.get(
```

Comment:
I'm not certain. It may be used for something within the class or its descendants. Either way, as it had it before I wanted to ensure it was still present, but that it wouldn't cause problems by effectively overriding the supplied username, password, etc, which would then still be used by _delete_amqp_queues (see lines below, comment for which will be updated when I reach that stage).